app/models/policy.rb

```diff
@@ -94,8 +94,23 @@ def destroy_orphaned_business_objective
     audit_bo_autoremove(removed_bos)
   end
 
+  def supported_os_minor_versions
+    profile_os_major_version = initial_profile.os_major_version
+    supported_benchmarks.map do |bm|
+      SupportedSsg.by_ssg_version[bm.version]
+                  .select { |ssg| ssg.os_major_version == profile_os_major_version }
+                  .map(&:os_minor_version)
+    end.flatten.uniq
```

Comment:
No need for the `uniq`, doesn't matter how many times the value is there if it's there.

```suggestion
    end.flatten
```